File: dotnet/windntrees.net/Application/Scripts/ViewModels/ProductFeature.js
function ProductFeature(data) {
    var instance = this;

    instance._datakey = data.UID;

    instance.UID = data.UID;

    instance.ProductID = data.ProductID;

    instance.Name = ko.observable(data.Name);
    instance.Name.extend({
        required: true,
        maxLength: 10
    });

    instance.Description = ko.observable(data.Description);
    instance.Description.extend({
        maxLength: 255
    });

    instance.RowVersion = data.RowVersion;
}

File: dotnet/windntrees.core/DataAccess.Core/Models/ProductFeature.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DataAccess.Core.Models
{
    public partial class ProductFeature
    {
        [Key]
        [Column("UID")]
        public Guid Uid { get; set; }
        [StringLength(255)]
        public string Description { get; set; }
        [Required]
        [StringLength(100)]
        public string Name { get; set; }
        [Column("ProductID")]
        public Guid ProductId { get; set; }
        public byte[] RowVersion { get; set; }

        [ForeignKey("ProductId")]
        [InverseProperty("ProductFeature")]
        public Product Product { get; set; }
    }
}
